client/src/client-only-routes/show-certification.tsx

```diff
@@ -193,25 +193,27 @@ const ShowCertification = (props: ShowCertificationProps): JSX.Element => {
     signedInUserName,
     location: { pathname }
   } = props;
+  const { pending, complete, errored } = fetchState;
+
+  useEffect(() => {
+    if (!isValidCert) {
+      createFlashMessage(certificateMissingErrorMessage);
+    } else if (!pending && errored) {
+      createFlashMessage(standardErrorMessage);
+    } else if (!pending && !complete && !errored) {
+      createFlashMessage(reallyWeirdErrorMessage);
+    }
+  }, [isValidCert, createFlashMessage, pending, errored, complete]);
 
   if (!isValidCert) {
```

Comment:
```suggestion
```

Reply:
If we don't check `isValidCert` here, the page will get stuck showing the Loader since `pending == true` by default.